frontend/src/components/Menu.js

```diff
@@ -1,46 +1,53 @@
-import React, { useEffect, useState } from 'react';
+import { useState } from 'react'
+import { useCacheFetch } from '../cache-hook'
 import Header from './Header';
 import Navbar from './Navbar';
-import axios from 'axios';
 import MealtimeSection from './MealtimeSection';
 import NotifyModal from './NotifyModal';
 import { SelectItemProvider } from './SelectItemContext';
 import Preloader from './Preloader';
 
 export default function Menu() {
-    const [menu, setMenu] = useState(null);
+    let menu = null;
+    let date = new Date('2022-01-18');
```

Comment:
can we have it use today instead of this random date? with `const date = new Date()`?

I'll give this a more thorough review later on, this is just one thing that brice was trying to debug

Reply:
Yee that's an easy fix! :)